package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lighthouse",
-  "version": "2.5.0",
+  "version": "2.5.1",
```

Comment:
need extension bump and changelog.md, too :)

Reply:
changelog updated. whitespace sorry